holoviews/plotting/renderer.py

```diff
@@ -529,6 +528,9 @@ def save(self_or_cls, obj, basename, fmt='auto', key={}, info={},
                 resources = CDN
             elif resources.lower() == 'inline':
                 resources = INLINE
+            if fmt not in ["html", "png", "auto"]:
+                raise ValueError("Format for exporting panel not supported: %s" % fmt)
+            basename = basename if fmt == "auto" else "%s.%s" % (basename, fmt)
```

Comment:
Suspect this should be something like this:

```suggestion
            fmt = 'html' if fmt == 'auto' else fmt
            if basename.split('.')[-1] not in ('html', 'png'):
                basename = "%s.%s" % (basename, fmt)
```

Reply:
`basename` is already stripped of the extension before being passed to `Renderer.save `.
Do you want me to add the if clause anyway?

If you want me to make explicit that "auto" corresponds to "html" I could also do something like this:
```python
fmt = 'html' if fmt == 'auto' else fmt
basename = "%s.%s" % (basename, fmt)
```